### PR TITLE
KYC selection page for KGLD 

### PR DESCRIPTION
**Pull request by moka1812**

### CODE REVIEW

**Comment by moka1812** [comment]
Screen Shot 2020-07-03 at 1 51 32 PM

**Comment by Git-on-my-level** [comment]
There's too much vertical space between the elements, can you narrow the space a bit more so it's more like the original
image

**Comment by moka1812** [comment]
Screen Shot 2020-07-03 at 4 17 20 PM

File: ts/kgld/Web/src/index.css [hunk 1]
```diff
@@ -90,7 +90,7 @@ button.outline {
 .container {
   min-width: 100%;
   min-height: 100%;
-  padding-top: 8rem;
+  padding-top: 6rem;
```
Comment (chauhannishith) [inline ts/kgld/Web/src/index.css:93]: don't make this change over here. Make it in the kycselection styled component as you made for the button

**Comment by Git-on-my-level** [comment]
This PR has been open for 2 weeks, why?

File: ts/kgld/Web/src/components/kyc/kyctypeselection.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import styled from 'styled-components'
+import { withRouter } from 'react-router-dom'
+import { ContainerRouterProps } from 'Web/src/interface'
+
+const KYCSelection = styled.div`
```
Comment (chauhannishith) [inline ts/kgld/Web/src/components/kyc/kyctypeselection.tsx:6]: move this css to blocks folder

**Pull request merged**